### DISCUSSION TITLE
QR code question

### DISCUSSION THREAD

**Discussion by ashapiro-fn** [pivpn/pivpn#1489] [Support]
Need a quick sanity check; I think there might be a bug with QR code generation. 

After creating a new client, the allowed ips are set to full tunnel, (0.0.0.0/0) by default. I go into the config for the client and change the allowed ips to my subnet to use split tunnel mode. 

I then generate a QR code for the client config, but the config encoded in that QR code doesn’t have my updated list of allowed ips, but the default 0.0.0.0/0. 

Can someone confirm if they’re seeing the same behavior? Am I doing something wrong?

**Comment by orazioedoardo** [comment]
When you change allowed IPs in the client config, you are just changing... the client config. If you want to change the template such that future clients have a different allowed IPs , edit `/etc/pivpn/wireguard/setupVars.conf`
